src/platform/w3dengine/client/w3ddisplay.cpp

```diff
@@ -933,9 +1673,594 @@ bool W3DDisplay::Is_LetterBox_Fading()
 // 0x0073E350
 int W3DDisplay::Get_Last_Frame_Draw_Calls()
 {
-#ifdef GAME_DLL
-    return Call_Method<int, W3DDisplay>(0x0073E350, this);
-#else
+    // return Debug_Statistics::Get_Draw_Calls();
     return 0;
+}
+
+void W3DDisplay::Update_Average_FPS()
+{
+#ifdef PLATFORM_WINDOWS
+    static uint64_t lastUpdateTime64;
+    static int historyOffset;
+    static int numSamples;
+    static double fpsHistory[30];
```

Comment:
All these static variables need to be zero initialized.

Reply:
Amusingly none of it is in original but yeah for safety should

Reply:
fixed.